airflow/contrib/hooks/gcs_hook.py

```diff
@@ -435,20 +348,13 @@ def get_crc32c(self, bucket, object):
         """
         self.log.info('Retrieving the crc32c checksum of '
                       'object: %s in bucket: %s', object, bucket)
-        service = self.get_conn()
-        try:
-            response = service.objects().get(
-                bucket=bucket,
-                object=object
-            ).execute()
-
-            crc32c = response['crc32c']
-            self.log.info('The crc32c checksum of %s is %s', object, crc32c)
-            return crc32c
-
-        except HttpError as ex:
-            if ex.resp['status'] == '404':
-                raise ValueError('Object Not Found')
+        client = self.get_conn()
+        bucket = client.get_bucket(bucket)
+        blob = bucket.blob(blob_name=object)
+        blob.reload()
+        blob_crc32c = blob.crc32c
+        self.log.info('The crc32c checksum of %s is %s', object, blob_crc32c)
+        return blob_crc32c
```

Comment:
The same comment as
https://github.com/apache/airflow/pull/5054/files#r272844679

Reply:
changed